python/python/lance/dataset.py

```diff
@@ -1280,6 +1281,15 @@ def create_scalar_index(
         replace : bool, default True
             Replace the existing index if it exists.
 
+        Optional Parameters
+        -------------------
+        with_position: bool, default True
+            This is for the ``INVERTED`` index. If True, the index will store the
+            positions of the words in the document, so that you can conduct phrase
+            query. This will significantly increase the index size.
+            It won't impact the performance of non-phrase queries even if it is set to
+            True.
```

Comment:
Do mean `False` here?
```suggestion
            It won't impact the performance of non-phrase queries even if it is set to
            False.
```

Reply:
It's `True`, we won't load the positions for non-phrase query, so it won't impact the non-phrase query performance

Reply:
Oh I see.